MTM-60479 - release notes for data broker operations delivery improvements (#2427)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-552-0-databroker-operations-delivery-improvements.md

---
date: 2024-09-11
title: "Data broker operations delivery improvements"
product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-V6J_FcOT2
    label: Data broker
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-60479
version: 10.20.552.0
---

> Data broker operations created on the destination tenant during network interruptions were not reliably forwarded to the source tenant. With the new fix, we have introduced a robust retry mechanism that ensures operations are delivered reliably, regardless of network stability. Users can now expect consistent data synchronization and improved reliability in data broker operations.

Reviewer: ```suggestion
Data broker operations created on the destination tenant were not reliably forwarded to the source tenant during network interruptions. With this change, a robust retry mechanism has been introduced that ensures operations are delivered reliably, regardless of network stability. Users can now expect consistent data synchronization and improved reliability in data broker operations.
```
